Apps/W1/HybridGP/app/src/Migration/Accounts/GPAccountMigrator.codeunit.al

```diff
@@ -232,36 +335,68 @@ codeunit 4017 "GP Account Migrator"
         GLAccDataMigrationFacade.ModifyGLAccount(true);
     end;
 
-    procedure GenerateGLTransactionBatches(GPAccount: Record "GP Account");
+    local procedure MigrateStatisticalAccountDetailsImp(GPAccount: Record "GP Account")
+    var
+        StatisticalAccount: Record "Statistical Account";
+        DataMigrationErrorLogging: Codeunit "Data Migration Error Logging";
+        AccountNum: Code[20];
+    begin
+        AccountNum := CopyStr(GPAccount.AcctNum.Trim(), 1, MaxStrLen(StatisticalAccount."No."));
+
+        if StatisticalAccount.Get(AccountNum) then
+            exit;
+
+        DataMigrationErrorLogging.SetLastRecordUnderProcessing(Format(GPAccount.RecordId));
+
+        StatisticalAccount.Init();
+        StatisticalAccount."No." := AccountNum;
+        StatisticalAccount.Name := CopyStr(GPAccount.Name, 1, MaxStrLen(StatisticalAccount.Name));
+        StatisticalAccount.Insert(true);
```

Comment:
I would recommend to set the global dimension 1 and 2 Code from the segments.

Reply:
Updates, thanks